package.json

```diff
@@ -24,7 +24,6 @@
     "broccoli-stew": "^1.5.0",
```

Comment:
I don't think the broccoli addons should be i deps either.

Reply:
doh, why github no refresh :/